### PR TITLE
Added helpful tips flag for system channels

### PR DESCRIPTION
**Pull request by TheOneMusic**
## Summary

Discord recently added the helpful tips notifications options for the system message channel. This pull request adds it in the [`SystemChannelFlags`](https://github.com/Rapptz/discord.py/blob/master/discord/flags.py#L140).

Screenshot 2021-04-14 at 19 38 05

## Checklist

- [x] If code changes were made then they have been tested.
    - [x] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: discord/http.py [hunk 1]
```diff
@@ -1193,6 +1193,8 @@ def create_interaction_response(self, interaction_id, token):
         )
         return self.request(r)
 
+    # Congratulations hunter, you found the secret message!
+
```
Comment (Zomatree) [inline discord/http.py:1197]:
```suggestion
```

**Comment by Rapptz** [comment]
As seen https://github.com/Rapptz/discord.py/issues/6605 the appropriate name for this is `guild_reminder_notifications`. This is also missing the versionadded.

**Comment by TheOneMusic** [comment]
@Rapptz Thank you. I made the changed

File: discord/flags.py [hunk 2]
```diff
@@ -199,6 +199,13 @@ def premium_subscriptions(self):
         """:class:`bool`: Returns ``True`` if the system channel is used for Nitro boosting notifications."""
         return 2
 
+    @flag_value
+    def guild_reminder_notifications(self):
+        """:class:`bool`: Returns ``True`` if the system channel is used for server setup helpful tips notifications.
+
+        .. versionadded:: 2.0"""
```
Comment (Rapptz) [inline discord/flags.py:206]:
```suggestion
        .. versionadded:: 2.0
        """
```

**Review by Rapptz — APPROVED** [review]

**Pull request merged**